hedera-node/infrastructure/grafana/dashboards/production/hedera-node/arbitrary-stat-prom.json

```
@@ -197,7 +212,7 @@
           "refId": "StandardVariableQuery"
         },
         "refresh": 1,
-        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|platform.*|merkle.*)/",
+        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|internal.*|platform.*)/",
```

```suggestion
        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|platform.*|merkle.*)/",
```

This was added in https://github.com/hashgraph/hedera-services/pull/8682/files to allow `merkle_*` metrics to show up in this dashboard.